packages/package-json-lint/src/lint.js

```diff
@@ -21,7 +21,7 @@ const lint = ({ packageJsonData, config }) => {
     const rule = rules[ruleId];
     const ruleConfig = getRuleConfig({ rule, ruleInformation });
     if (ruleConfig.severity !== 'off') {
-      return rule.create({ ruleConfig, report: issue => issues.push(issue) });
+      return rule.create({ ruleConfig, type: config.type, report: issue => issues.push(issue) });
```

Comment:
Let's default the type here rather than in each individual rule.

Reply:
Updated here https://github.com/cerner/terra-toolkit/pull/700/commits/8ad7e5e9e43dce7f25ad36999a0395157ccbb5e1